src/test/domainFactories.ts

```diff
@@ -18,17 +18,24 @@ import { ConfidenceLevel } from '../domain/model/test/ConfidenceLevel';
 import { InterpretationRules } from '../domain/model/test/interpretation/InterpretationRules';
 import { InterpretationTheme } from '../domain/model/test/interpretation/Interpretation';
 import { ADD_TAKE_HOME_TEST_RESULT } from '../domain/model/authentication/Permissions';
+import { AuthenticationDetails } from '../domain/model/user/AuthenticationDetails';
+import { AuthenticationMethod } from '../domain/model/user/AuthenticationMethod';
+import { AuthenticationValue } from '../domain/model/user/AuthenticationValue';
 
 export function aNewUser() {
-  return new User(new UserId(), anEmail());
+  return User.create(magicLinkAuthenticationDetails());
+}
+
+export function magicLinkAuthenticationDetails(email = `${v4()}@example.com`) {
```

Comment:
Why break the naming convention

Reply:
`aMagicLinkAuthentication`?

Reply:
Can do, just felt weird a bit

Reply:
I wonder if you could switch it to be `anAuthenticationDetails(type, identifier)`, which would be more akin to what we do with `aRoleWithPermissions`.  At the moment I find the consumption of this method a bit uncomfortable.  The name is definitely contributing to that.

Reply:
Actually, maybe just ditch this thing.  If you do what I suggested it offer no real value over using the class directly.  Even how you have it now, it offers minimal value over using the class directly (only omitting the type).  So maybe we don't need the factory at all...

Reply:
We added it later on, as the tests got quite verbose without it (`new AuthenticationDetails(AuthenticationMethod.MAGIC_LINK, new AuthenticationValue('mail@example.com'))`).